src/components/assets/AssetPreview.vue

```diff
@@ -223,4 +223,8 @@ export default {
   margin-right: 5px;
 }
 
+.assetlibrary-item-preview-outer {
+  text-align: center;
+}
+
```

Comment:
@pauline2k - Fyi, there is built-in style `text-center`: https://vuetifyjs.com/en/styles/text-and-typography/